### ISSUE TITLE
PlusApp development release not working

### ISSUE THREAD

**Issue by jamesobutler** [PlusToolkit/PlusLib#594]
I downloaded the most recent PlusApp development releases (PlusApp-2.9.0.20190903 - 32/64 bit versions) and upon attempting to start PlusServer.exe or PlusVersion.exe it just exits without outputting anything.

```log
PS C:\Users\JamesButler\PlusApp-2.9.0.20190903-Win32\bin> ./PlusServer.exe --config-file="C:\Users\JamesButler\PlusApp-2.9.0.20190903-Win32\config\PlusDeviceSet_Server_MmfColorVideoCapture.xml"
PS C:\Users\JamesButler\PlusApp-2.9.0.20190903-Win32\bin>
PS C:\Users\JamesButler\PlusApp-2.9.0.20190903-Win32\bin> ./PlusVersion.exe
PS C:\Users\JamesButler\PlusApp-2.9.0.20190903-Win32\bin>
```

**Comment by jamesobutler** [comment]
My system is Windows 10 Pro running build 1903

I also have these redistributables installed
![redistributable](https://user-images.githubusercontent.com/15837524/64449692-8671c280-d0ae-11e9-8861-eb6eebbb76c0.PNG)

**Comment by Sunderlandkyl** [comment]
I just tried the latest nightly and got an error about a missing OpenCV dll.
I'll look into it.

**Comment by jamesobutler** [comment]
I was going to look if there were errors with the build on the CDASH dashboard, but the linked pages on https://plustoolkit.github.io/developersguide.html  are giving the following as seen below.  Is the dashboard down or are the links just outdated? 
![cdash_error](https://user-images.githubusercontent.com/15837524/64450231-abb30080-d0af-11e9-9fbd-1c1b53d6437f.PNG)

**Comment by Sunderlandkyl** [comment]
Unfortunately the dashboard has been down for a while. I've asked the IT staff several times to look at it, but it hasn't been fixed yet.

**Comment by Sunderlandkyl** [comment]
Just ran local tests on the build machine, and they are passing, so there must be a problem with packaging.

**Comment by jamesobutler** [comment]
> Unfortunately the dashboard has been down for a while. I've asked the IT staff several times to look at it, but it hasn't been fixed yet.

On that page it is saying the CDash version is 1.6.4 which was released in 4-16-2010.  Something that old probably has a good chance of failing haha. 

Looks like stable is version 2.6.0 released 11-16-2018.  Looks like Slicer is even using the development version as it is showing version 2.7.0 http://slicer.cdash.org/index.php?project=SlicerPreview.

**Comment by Sunderlandkyl** [comment]
Yeah, initial issue was an incompatible PHP version. That was fixed, but we were left with the pages as you see them now. I suggested that IT could upgrade the CDash version, but nothing has happened yet.

**Comment by Sunderlandkyl** [comment]
For some reason, it looks like the OpenCV dlls are being installed in the wrong directory.

They should be in:
```PLUS_INSTALL_DIR\bin```

But instead they're in one of these paths:
```PLUS_INSTALL_DIR\x86```
```PLUS_INSTALL_DIR\x64```

Does copying the dll to the correct location fix the problem for you?

**Comment by jamesobutler** [comment]
There is no /x86 or /x64 directory using the installer.  I don't currently have a Plus build that specifies to build OpenCVVideo.

**Comment by Sunderlandkyl** [comment]
The nightly builds should now work again.
Changes in the OpenCV 3.4 branch resulted in a change in the dll installation directory.

Fixed by changing the OpenCV version to the 3.4.7 tag and re-configuring the nightly builds.
https://github.com/PlusToolkit/PlusBuild/commit/4be18ef49e74d6c73cff9ae6f612925a26a8552c
